test/cli.js

```diff
@@ -680,6 +680,49 @@ test('legacy snapshot files are reported to the console', t => {
 	});
 });
 
+test('snapshots infer their location from sourcemaps', t => {
+	t.plan(8);
+	const dirname = path.join('fixture/snapshots/test-sourcemaps');
+	const testLocationTypes = [
+		'src',
+		'src/test/snapshots',
+		'src/feature/__tests__/__snapshots__'
+	];
+	const removeExists = relFilePath => {
```

Comment:
This doesn't remove anything though?

You should remove these files at least prior to running the test, otherwise on local setups you won't be able to tell if the code has broken if they're persisted from previous runs.